src/main/java/org/jabref/logic/bibtex/BibDatabaseMerger.java

```diff
@@ -0,0 +1,65 @@
+package org.jabref.logic.bibtex;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.database.BibDatabaseModeDetection;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibEntryTypesManager;
+import org.jabref.model.entry.BibtexString;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BibDatabaseMerger {
```

Comment:
`DatabaseMerger` should be sufficient...I have actually no idea why we added the prefix bib in some instances...